.github/workflows/ci.yaml

```diff
@@ -0,0 +1,88 @@
+name: Continuous Integration
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/ci.yml
+      - "oculy/**"
+      - pyproject.toml
+
+jobs:
+  formatting:
+    name: Check code formatting
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install tools
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r lint_requirements.txt
+      - name: Isort
+        run: |
+          isort oculy -c;
+      - name: Black
+        if: always()
+        run: |
+          black oculy --check;
+      - name: Flake8
+        if: always()
+        run: |
+          flake8 oculy;
+  tests:
+    name: Unit tests
+    runs-on: ${{ matrix.os }}
+    needs:
+      - formatting
+    if: needs.formatting.result == 'success'
+    strategy:
+      matrix:
+        os: [windows-latest]
+        python-version: ["3.9", "3.10"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r test_requirements.txt
+      - name: Install project
+        run: |
+          pip install -e .
```

Comment:
It is better to test against an installed version of the package so you can remove the -e

Reply:
👍